docs/tutorial.dox

```diff
@@ -37,55 +39,93 @@ A Galois user program consists of operators, schedules and data structure API ca
 
-When defining a galois::graphs::LC_CSR_Graph, you must provide as template parameters NodeTy, the type of data stored on each node; and EdgeTy, the type of data stored on each edge. Use void when no data needs to be stored on nodes or edges. See galois::graphs::LC_CSR_Graph for other optional template parameters.
+We will focus on galois::graphs::LC_CSR_Graph in this section; @ref galois_morph_graph_api are discussed later. When defining a galois::graphs::LC_CSR_Graph, you must provide as template parameters `NodeTy`, the type of data stored on each node; and `EdgeTy`, the type of data stored on each edge. Use `void` when no data needs to be stored on nodes or edges. See galois::graphs::LC_CSR_Graph for other optional template parameters.
 
-Below is an example of defining an LC_CSR_Graph with an integer as both its node data type and edge data type:
+Below is an example of defining an LC_CSR_Graph with `int` as both its node data type and edge data type:
 
 @snippet lonestar/tutorial_examples/GraphTraversalSerial.cpp Define LC_CSR_Graph
 
-The following code snippet shows how to instantiate and read in a graph from a file (in binary gr format):
+The following code snippet shows how to instantiate and read in a graph from a file (in binary .gr format):
 
 @snippet lonestar/tutorial_examples/GraphTraversalSerial.cpp Read a graph
 
-To access graph elements, use the following constructs.
-<ol>
-<li> Iteration over nodes: use the node iterator galois::graphs::LC_CSR_Graph::iterator given by galois::graphs::LC_CSR_Graph::begin and galois::graphs::LC_CSR_Graph::end.
-<li> Iteration over outgoing edges of a node: use the edge iterator galois::graphs::LC_CSR_Graph::edge_iterator given by galois::graphs::LC_CSR_Graph::edge_begin and galois::graphs::LC_CSR_Graph::edge_end.
-<li> To read/write a node data: use galois::graphs::LC_CSR_Graph::getData.
-<li> To read/write an edge data: use galois::graphs::LC_CSR_Graph::getEdgeData.
-<li> To access the destination node of an outgoing edge: use galois::graphs::LC_CSR_Graph::getEdgeDst.
-</ol>
+To access graph elements, use the following constructs:
+<ul>
+<li> **To iterate over nodes:** use the node iterator galois::graphs::LC_CSR_Graph::iterator given by galois::graphs::LC_CSR_Graph::begin and galois::graphs::LC_CSR_Graph::end.
+<li> **To iterate over outgoing edges of a node:** use the edge iterator galois::graphs::LC_CSR_Graph::edge_iterator given by galois::graphs::LC_CSR_Graph::edge_begin and galois::graphs::LC_CSR_Graph::edge_end.
+<li> **To access data of a node:** use galois::graphs::LC_CSR_Graph::getData.
```

Comment:
Minor nitpick here, it should be "To access the data of..." for both a node and an edge. This is still much better than it was before.